cpp/proto/serializer.cc

```diff
@@ -443,44 +348,20 @@ SerializeResult TLSSerializer::WriteList(const repeated_string& in,
   size_t prefix_length = PrefixLength(max_total_length);
   CHECK_GE(length, prefix_length);
 
-  WriteUint(length - prefix_length, prefix_length);
+  WriteUint(length - prefix_length, prefix_length, output);
```

Comment:
Hmm, not sure, but just to make sure: this one is *correct* to not clear `output`, right?